app/helpers/strategies_helper.rb

```diff
@@ -28,99 +28,20 @@ def edit_strategy_props(strategy, viewers)
     [
-      {
-        id: 'strategy_name',
-        type: 'text',
-        name: 'strategy[name]',
-        label: t('common.name'),
-        value: strategy.name || nil,
-        placeholder: t('strategies.form.name_hint'),
-        required: true,
-        dark: true
-      },
-      {
-        id: 'strategy_description',
-        type: 'textarea',
-        name: 'strategy[description]',
-        label: t('strategies.form.describe'),
-        value: strategy.description || nil,
-        required: true,
-        dark: true
-      },
-      {
-        id: 'strategy_category',
-        type: 'quickCreate',
-        name: 'strategy[category][]',
-        label: t('categories.plural'),
-        placeholder: t('common.form.press_enter'),
-        checkboxes: category_checkboxes,
-        formProps: quick_create_category_props
-      },
+      build_strategy_name(strategy),
+      build_strategy_description(strategy),
+      build_strategy_category,
       get_viewers_input(viewers, 'strategy', 'strategies', strategy),
-      {
-        id: 'strategy_comment',
-        type: 'switch',
-        name: 'strategy[comment]',
-        label: t('comment.allow_comments'),
-        value: true,
-        uncheckedValue: false,
-        checked: strategy.comment,
-        info: t('comment.hint'),
-        dark: true
-      },
-      {
-        id: 'strategy_publishing',
-        type: 'switch',
-        label: t('strategies.form.draft_question'),
-        dark: true,
-        name: 'publishing',
-        value: '0',
-        uncheckedValue: '1',
-        checked: !strategy.published?
-      },
-      {
-        id: 'strategy_perform_strategy_reminder',
-        type: 'checkbox',
-        name: 'strategy[perform_strategy_reminder_attributes][active]',
-        label: t('common.daily_reminder'),
-        info: t('strategies.form.daily_reminder_hint'),
-        value: true,
-        uncheckedValue: false,
-        checked: strategy&.perform_strategy_reminder&.active,
-        dark: true
-      },
-      {
-        id: 'strategy_perform_strategy_reminder_attributes_id',
-        name: 'strategy[perform_strategy_reminder_attributes][id]',
-        type: 'hidden',
-        value: strategy&.perform_strategy_reminder&.id
-      }
+      build_strategy_comment(strategy),
+      build_strategy_publishing(strategy),
+      build_strategy_reminder(strategy).merge(type: 'checkbox'),
+      build_strategy_reminder_attributes(strategy)
     ]
   end
-  # rubocop:enable MethodLength
 
   def quick_create_strategy_form_inputs
-    strategy_form = strategy_form_inputs(@strategy, @viewers)
-    [
-      strategy_form[0],
-      strategy_form[1]
-    ]
-  end
-
-  def category_checkboxes
-    checkboxes = []
-    @categories.each do |item|
-      checkboxes.push(
-        id: item.slug,
-        label: item.name,
-        value: item.id,
-        checked: @strategy.category.include?(item.id)
-      )
-    end
-    checkboxes
+    strategy_form_inputs(@strategy, @viewers)[0..1]
```

Comment:
I think this will be more readable if it is 
```suggestion
    [
        build_strategy_name(@strategy),
        build_strategy_description(@strategy)
    ]
```
@julianguyen Correct me if I'm wrong

Reply:
Whoa embedded code snippets - that's new
Agreed, I think it's readable the way @hmasila suggested!

Reply:
Thanks for the feedback! I will make these changes in the next days